Marharyta_Miliuk/Module 3 - Java/Week 2/NewYearPresent/src/by/epam/cdp/java/nypresent/Present.java

```diff
@@ -0,0 +1,53 @@
+package by.epam.cdp.java.nypresent;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import by.epam.cdp.java.nypresent.beans.*;
+import by.epam.cdp.java.nypresent.utils.InputOutputStreams;
+
+
+public class Present{
+	
+	public ArrayList<Sweets> collection = new ArrayList<Sweets>();//creating public field for collection
+	
+	public Comparator<Sweets> SweetsNameComparator = new Comparator<Sweets>() {//creating and defining comparator field to use it in the sorting
+		
+	    public int compare(Sweets candy1, Sweets candy2) {
+
+	    String candyName1 = candy1.getName().toUpperCase();
+	    String candyName2 = candy2.getName().toUpperCase();
+
+	    return candyName1.compareTo(candyName2);
+	    }
+	 };
+	
+	public void sortCollectionByName(){//creating method to sort candies in the present by names alphabetically
+		
+	    Collections.sort(collection, SweetsNameComparator);
+	         for (Sweets e:collection){
```

Comment:
I recommend perform only sorting in the method. Need to print the collection if needed in main method